TimberAPI/TimberAPIPlugin.cs

```diff
@@ -15,9 +16,12 @@ namespace TimberbornAPI.Internal
     public class TimberAPIPlugin : BaseUnityPlugin
     {
         public static string Guid = "com.timberapi.timberapi";
-        
+        internal static ManualLogSource Log;
+
         public void Awake()
         {
+            Log = Logger;
```

Comment:
We should do a cleanup at some point to be consistent with our logging

Reply:
> We should do a cleanup at some point to be consistent with our logging

Let's make another PR for that specialized in a logging feature.